x/merkledb/proof_test.go

```diff
@@ -263,49 +352,75 @@ func Test_RangeProof_MaxLength(t *testing.T) {
 	require.ErrorIs(err, ErrInvalidMaxLength)
 }
 
-func Test_Proof(t *testing.T) {
+func Test_Proof_Path(t *testing.T) {
 	require := require.New(t)
 
 	dbTrie, err := getBasicDB()
 	require.NoError(err)
 	require.NotNil(dbTrie)
+
+	var (
+		key  = []byte{0x6B, 0x65, 0x79}
+		key0 = []byte{0x6B, 0x65, 0x79, 0x30}
+		key1 = []byte{0x6B, 0x65, 0x79, 0x31}
+		key2 = []byte{0x6B, 0x65, 0x79, 0x32}
+		key3 = []byte{0x6B, 0x65, 0x79, 0x33}
+		key4 = []byte{0x6B, 0x65, 0x79, 0x34}
```

Comment:
Can we just use strings? These bytes values are hard to reason about/look ugly... and chars are 1 byte each + byte representations are visible in the debugger anyways.